tests/integration/catalogue/test_options.py

```diff
@@ -32,7 +32,7 @@ def test_queryset_per_product(self):
         qs = Product.objects.browsable().base_queryset().filter(id=self.product.id)
         product = qs.first()
         self.assertTrue(product.has_options)
-        self.assertTrue(product.has_product_options, 1)
+        self.assertEqual(product.has_product_options, 1)
```

Comment:
Unfortunately has_product_options will return a boolean and not an integer. So this does not fix the test, in fact it breaks the test! While you have a point that this test is faulty, this is not a good fix, it is not even a fix at all!

Reply:
apologies for wasting you time then. I scanned a few hundred codebases to find cases of what _looks_ like misuse of `assertTrue`. False positives will happen. Sorry it was your repo that had the false positive :)